app/controlplane/Makefile

```diff
@@ -10,6 +10,8 @@ config: check-buf-tool
 api: check-buf-tool
 	cd ./plugins/sdk/v1/plugin/api && buf generate
 	cd ./api && buf generate
+	# Remove all json schema that are not part of the workflowcontract since we don't need them
+	find ./api/gen/jsonschema -type f ! -name 'workflowcontract.*' -exec rm {} +
```

Comment:
mm, I am not sure this is correct. in the CI we might be running generation without makefile, and afaik we are not enforcing using `make` for things

What's wrong with having all the files? How many are those?

Reply:
There are 189 generated files without the option of not generating them.